### PR TITLE
feat: add runId to slack thread when sanities fail

### PR DESCRIPTION
**Pull request by dingxuanyao**
https://tophat.atlassian.net/jira/software/c/projects/WEB/boards/247?modal=detail&selectedIssue=SERVINFRA-1209

### CODE REVIEW

File: service/src/alertOnResult.js [hunk 1]
```diff
@@ -186,6 +186,15 @@ const sendSlackMessage = async function (
...
                 }
 
+                if (message.runId) {
+                    await slack.chat.postMessage({
+                        channel: channel,
+                        thread_ts: thread,
+                        title: 'runId for sanity test:',
+                        text: message.runId,
+                    })
+                }
```
Comment (noahnu) [inline service/src/alertOnResult.js:196]: I think this is perfectly fine to merge as is, though something I'd like to do when my [larger PR](https://github.com/tophat/sanity-runner/pull/269) is merged in is consolidate all the slack messages into a single message for improved readability and to reduce chance of rate limiting.

Comment (dingxuanyao) [reply service/src/alertOnResult.js:196]: Agreed, I don't love the thread either.
Created story: https://tophat.atlassian.net/browse/SERVINFRA-1246

**Comment by noahnu** [comment]
You'll want to rebase

File: service/src/alerts/index.ts [hunk 2]
```diff
@@ -12,10 +12,12 @@ export async function alertOnResult({
     testFiles,
     results,
     testVariables,
+    runId,
 }: {
     testFiles: Record<string, string>
     results: EnhancedAggregatedResult
     testVariables: Partial<Record<string, string>>
+    runId: string,
```
Comment (noahnu) [inline service/src/alerts/index.ts:20]:
```suggestion
    runId: string
```

L20 is a typescript type which uses semicolons or newlines as separators. So you can just remove the comma

Comment (dingxuanyao) [reply service/src/alerts/index.ts]: Thanks @noahnu 🙏 , test is passing now.

**Review by noahnu — APPROVED** [review]
LGTM until we can consolidate the messages

**Pull request merged**